Bottom: 97526e88d83bffe732c5e4d39f88fd67300719b6
Top:    2be48773d9638d109d7ec01fb7dedea530871d42
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-06 16:02:56 -0500

Refresh of create-role

---
diff --git a/ansible-playbook/group_vars/all/nexus3.yml b/ansible-playbook/group_vars/all/nexus3.yml
index e4b17643..b82a2277 100644
--- a/ansible-playbook/group_vars/all/nexus3.yml
+++ b/ansible-playbook/group_vars/all/nexus3.yml
@@ -1,133 +1,133 @@
-      nexus_version: 3.13.0
-      nexus_networks:
-        - name: '{{ docker_network_frontend }}'
-      docker: true
-      nexus_default_port: 80
-      public_hostname: '{{ nexus_hostname }}'
+nexus_version: 3.13.0
+nexus_networks:
+  - name: '{{ docker_network_frontend }}'
+docker: true
+nexus_default_port: 80
+public_hostname: '{{ nexus_hostname }}'
 
-      nexus_email_server_enabled: true
-      nexus_email_server_host: "mail.novatech-llc.com"
-      nexus_email_server_port: 587
-      nexus_email_server_username: "wiki"
-      nexus_email_server_password: "{{ nexus_email_password }}"
-      nexus_email_from_address: "nexus@novatech-llc.com"
-      nexus_email_subject_prefix: "[NEXUS]"
-      nexus_email_tls_enabled: true
-      nexus_email_tls_required: true
+nexus_email_server_enabled: true
+nexus_email_server_host: "mail.novatech-llc.com"
+nexus_email_server_port: 587
+nexus_email_server_username: "wiki"
+nexus_email_server_password: "{{ nexus_email_password }}"
+nexus_email_from_address: "nexus@novatech-llc.com"
+nexus_email_subject_prefix: "[NEXUS]"
+nexus_email_tls_enabled: true
+nexus_email_tls_required: true
 
-      nexus_ldap_realm: true
-      ldap_connections:
-        - ldap_name: 'NovaTech LDAP' # used as a key to update the ldap config
-          ldap_protocol: 'ldap' # ldap or ldaps
-          ldap_hostname: 'ldap.novatech-llc.com'
-          ldap_port: 389
-          ldap_auth: 'simple'
-          ldap_auth_username: 'cn=proxyagent,dc=novatech'
-          ldap_auth_password: '{{ openldap_proxyagent_password }}'
-          ldap_search_base: 'dc=novatech'
-          ldap_user_base_dn: 'ou=user'
-          #ldap_user_filter: '(cn=*)' # (optional)
-          ldap_user_object_class: 'inetOrgPerson'
-          ldap_user_id_attribute: 'uid'
-          ldap_user_real_name_attribute: 'cn'
-          ldap_user_email_attribute: 'mail'
-          ldap_group_base_dn: 'ou=group'
-          ldap_group_object_class: 'posixGroup'
-          ldap_group_id_attribute: 'gidNumber'
-          ldap_group_member_attribute: 'memberUid'
-          ldap_group_member_format: '${username}'
+nexus_ldap_realm: true
+ldap_connections:
+  - ldap_name: 'NovaTech LDAP' # used as a key to update the ldap config
+    ldap_protocol: 'ldap' # ldap or ldaps
+    ldap_hostname: 'ldap.novatech-llc.com'
+    ldap_port: 389
+    ldap_auth: 'simple'
+    ldap_auth_username: 'cn=proxyagent,dc=novatech'
+    ldap_auth_password: '{{ openldap_proxyagent_password }}'
+    ldap_search_base: 'dc=novatech'
+    ldap_user_base_dn: 'ou=user'
+    #ldap_user_filter: '(cn=*)' # (optional)
+    ldap_user_object_class: 'inetOrgPerson'
+    ldap_user_id_attribute: 'uid'
+    ldap_user_real_name_attribute: 'cn'
+    ldap_user_email_attribute: 'mail'
+    ldap_group_base_dn: 'ou=group'
+    ldap_group_object_class: 'posixGroup'
+    ldap_group_id_attribute: 'gidNumber'
+    ldap_group_member_attribute: 'memberUid'
+    ldap_group_member_format: '${username}'
 
-      nexus_privileges:
-        - name: all-repos-read  # used as key to update a privilege
-          description: 'Read & Browse access to all repos'
-          repository: '*'
-          actions:
-            - read
-            - browse
+nexus_privileges:
+  - name: all-repos-read  # used as key to update a privilege
+    description: 'Read & Browse access to all repos'
+    repository: '*'
+    actions:
+      - read
+      - browse
 
-      nexus_roles:
-        - id: Developers  # can map to a LDAP group id, also used as a key to update a role
-          name: developers
-          description: All developers
-          privileges:
-            - nx-search-read
-            - all-repos-read
-          roles: []
+nexus_roles:
+  - id: Developers  # can map to a LDAP group id, also used as a key to update a role
+    name: developers
+    description: All developers
+    privileges:
+      - nx-search-read
+      - all-repos-read
+    roles: []
 
-      nexus_local_users:
-        - username: buildbot # used as key to update
-          first_name: NovaTech
-          last_name: CI
-          email: buildbot@novatech-llc.com
-          password: "{{ nexus_buildbot_password }}"
-          roles:
-            - Developers # role ID here
+nexus_local_users:
+  - username: buildbot # used as key to update
+    first_name: NovaTech
+    last_name: CI
+    email: buildbot@novatech-llc.com
+    password: "{{ nexus_buildbot_password }}"
+    roles:
+      - Developers # role ID here
 
-      nexus_delete_default_blobstore: false
-      nexus_blob_split: false     # True - blobstores per format
+nexus_delete_default_blobstore: false
+nexus_blob_split: false     # True - blobstores per format
 
-      nexus_delete_default_repos: true
+nexus_delete_default_repos: true
 
-      nexus_config_maven: false
+nexus_config_maven: false
 
-      nexus_config_pypi: true
-      nexus_repos_pypi_hosted:
-        - name: pypi-hosted
-          write_policy: allow  # one of "allow", "allow_once" or "deny"
-      nexus_repos_pypi_proxy:
-        - name: pypi-proxy
-          remote_url: 'https://pypi.python.org/'
-      nexus_repos_pypi_group:
-        - name: pypi
-          member_repos:
-            - pypi-hosted
-            - pypi-proxy
+nexus_config_pypi: true
+nexus_repos_pypi_hosted:
+  - name: pypi-hosted
+    write_policy: allow  # one of "allow", "allow_once" or "deny"
+nexus_repos_pypi_proxy:
+  - name: pypi-proxy
+    remote_url: 'https://pypi.python.org/'
+nexus_repos_pypi_group:
+  - name: pypi
+    member_repos:
+      - pypi-hosted
+      - pypi-proxy
 
-      nexus_config_docker: true
-      nexus_repos_docker_hosted:
-        - name: docker-hosted
-          http_port: "{{ nexus_docker_hosted_port }}"
-          v1_enabled: True
-      nexus_repos_docker_proxy:
-        - name: docker-proxy
-          http_port: "{{ nexus_docker_proxy_port }}"
-          v1_enabled: True
-          index_type: "HUB"
-          proxy_url: "https://registry-1.docker.io"
-          use_nexus_certificates_to_access_index: false
-      nexus_repos_docker_group:
-        - name: docker
-          http_port: "{{ nexus_docker_group_port }}"
-          v1_enabled: True
-          member_repos:
-            - docker-hosted
-            - docker-proxy
+nexus_config_docker: true
+nexus_repos_docker_hosted:
+  - name: docker-hosted
+    http_port: "{{ nexus_docker_hosted_port }}"
+    v1_enabled: True
+nexus_repos_docker_proxy:
+  - name: docker-proxy
+    http_port: "{{ nexus_docker_proxy_port }}"
+    v1_enabled: True
+    index_type: "HUB"
+    proxy_url: "https://registry-1.docker.io"
+    use_nexus_certificates_to_access_index: false
+nexus_repos_docker_group:
+  - name: docker
+    http_port: "{{ nexus_docker_group_port }}"
+    v1_enabled: True
+    member_repos:
+      - docker-hosted
+      - docker-proxy
 
-      nexus_config_raw: true
-      nexus_repos_raw_hosted:
-        - name: download
-          strict_content_validation: false
-          write_policy: allow  # one of "allow", "allow_once" or "deny"
-        - name: ntel-ipkg
-          strict_content_validation: false
-          write_policy: allow  # one of "allow", "allow_once" or "deny"
-        - name: ntel-sstate
-          strict_content_validation: false
-          write_policy: allow  # one of "allow", "allow_once" or "deny"
-        - name: ptxdist-ipkg
-          strict_content_validation: false
-          write_policy: allow  # one of "allow", "allow_once" or "deny"
+nexus_config_raw: true
+nexus_repos_raw_hosted:
+  - name: download
+    strict_content_validation: false
+    write_policy: allow  # one of "allow", "allow_once" or "deny"
+  - name: ntel-ipkg
+    strict_content_validation: false
+    write_policy: allow  # one of "allow", "allow_once" or "deny"
+  - name: ntel-sstate
+    strict_content_validation: false
+    write_policy: allow  # one of "allow", "allow_once" or "deny"
+  - name: ptxdist-ipkg
+    strict_content_validation: false
+    write_policy: allow  # one of "allow", "allow_once" or "deny"
 
-      nexus_repos_raw_proxy:
-        - name: ubuntu-archive
-          remote_url: 'http://archive.ubuntu.com/ubuntu/'
-          strict_content_validation: false
+nexus_repos_raw_proxy:
+  - name: ubuntu-archive
+    remote_url: 'http://archive.ubuntu.com/ubuntu/'
+    strict_content_validation: false
 
-      nexus_repos_raw_group: []
+nexus_repos_raw_group: []
 
-      nexus_config_rubygems: false
-      nexus_config_bower: false
-      nexus_config_npm: false
-      nexus_config_nuget: false
-      nexus_config_gitlfs: false
-      nexus_config_yum: false
+nexus_config_rubygems: false
+nexus_config_bower: false
+nexus_config_npm: false
+nexus_config_nuget: false
+nexus_config_gitlfs: false
+nexus_config_yum: false
